src/interpreter/value.cc

```diff
@@ -325,6 +325,31 @@ namespace verona::interpreter
     }
   }
 
+  void FieldValue::add_isos(rt::ObjectStack& stack) const
+  {
+    switch (tag)
+    {
+      case Value::ISO:
```

Comment:
Why not have three cases here: `ISO`, `COWN_OWNED` and `default`?

Reply:
Most of the prototype compiler doesn't use `default` so that adding new states to the enums trigger compiler errors to review what should be done in the switch.  I was matching the style of the file.